src/controllers/bulk/bulkcontroller.cpp

```diff
@@ -35,22 +32,20 @@ void BulkReader::run() {
         // Blocked polling: The only problem with this is that we can't close
         // the device until the block is released, which means the controller
         // has to send more data
-        //result = hid_read_timeout(m_pHidDevice, data, 255, -1);
+        // result = hid_read_timeout(m_pHidDevice, data, 255, -1);
```

Comment:
Aha, here's an example where it added a space on a code item. But I guess we acknowledged that possibility.